src/components/WorkflowDetail.js

```
@@ -1,13 +1,13 @@
 import React, {useEffect, useState} from "react";
 import './WorkflowDetail.css'
 
-export default function WorkflowDetail({jobId}) {
+export default function WorkflowDetail({caseId}) {
```

ESLint

[error] 'caseId' is defined but never used.

(no-unused-vars)